js/src/personal/maliha/Maliha.module.css

```diff
@@ -34,30 +34,39 @@
 .menuItems{
   padding: 100px 0px 20px 120px;
   font-size: 40px;
-  font-weight: 100;
+  font-weight: 400;
   letter-spacing: 2px;
 }
 .modal {
   :global(.mantine-Modal-title){
-    padding: 50px 0px 20px 120px;
-    font-size: 50px;
-    font-weight: 150;
+    padding: 50px 0px 0px 100px;
+    font-size: 75px;
+    font-weight: 400;
+    font-family: "Handlee", Sans-Serif;
```

Comment:
fyi: if you want this font to be the default in your project, then you can add it to the mantine theme